src/hooks/useClickOutside/index.ts

```diff
@@ -14,7 +22,7 @@ export default function useOnClickOutside({ ref, callback }) {
     [ref, callback]
   );
 
-  const documentRef = useRef(document);
+  const documentRef = useRef(document.body);
```

Comment:
This doesn't affect functionality at all?

Reply:
@sahariko no it doesn't